### PR TITLE
Adds support for Exeed icons referenced via platform uris

### PR DESCRIPTION
**Pull request by arcanefoam**
Applies the patch provided in the Eclipse bugzilla

Cleaned up some deprecated code

### CODE REVIEW

File: plugins/org.eclipse.epsilon.dt.exeed/src/org/eclipse/epsilon/dt/exeed/ExeedImageTextProvider.java [hunk 1]
```diff
@@ -89,13 +90,25 @@ public void setShowStructuralInfo(boolean showStructuralInfo) {
...
-			imageDescriptor = plugin.getImageDescriptor("icons/" + icon + ".png");
+		// icon might be a platform:/resource uri
+		if (icon != null) {
+			if (icon.startsWith ("platform:/")) {
+				try {
+					URI iconuri = URI.createURI(icon, false);
+					imageDescriptor = plugin.getImageDescriptor(iconuri);
+				}
+				catch (IllegalArgumentException ex) {
+					// icon is not a valid uri
```
Comment (kolovos) [inline plugins/org.eclipse.epsilon.dt.exeed/src/org/eclipse/epsilon/dt/exeed/ExeedImageTextProvider.java:101]: The exception should be logged using LogUtil.log(e);

File: plugins/.project
Comment (kolovos) [inline plugins/.project:1]: This change is unrelated to the purpose of the PR. Remove?

File: plugins/.settings/org.eclipse.core.resources.prefs
Comment (kolovos) [inline plugins/.settings/org.eclipse.core.resources.prefs:1]: This change is unrelated to the purpose of the PR. Remove?

File: plugins/org.eclipse.epsilon.workflow.emf/ant/org/eclipse/epsilon/workflow/tasks/emf/antlib.xml
Comment (kolovos) [inline plugins/org.eclipse.epsilon.workflow.emf/ant/org/eclipse/epsilon/workflow/tasks/emf/antlib.xml:1]: This change is unrelated to the purpose of the PR. Remove?

File: plugins/org.eclipse.epsilon.workflow.emf/ant/org/eclipse/epsilon/workflow/tasks/emf/tasks.xml
Comment (kolovos) [inline plugins/org.eclipse.epsilon.workflow.emf/ant/org/eclipse/epsilon/workflow/tasks/emf/tasks.xml:1]: This change is unrelated to the purpose of the PR. Remove?

File: plugins/org.eclipse.epsilon.workflow/ant/org/eclipse/epsilon/workflow/tasks/antlib.xml
Comment (kolovos) [inline plugins/org.eclipse.epsilon.workflow/ant/org/eclipse/epsilon/workflow/tasks/antlib.xml:1]: This change is unrelated to the purpose of the PR. Remove?

File: plugins/org.eclipse.epsilon.workflow/ant/org/eclipse/epsilon/workflow/tasks/tasks.xml
Comment (kolovos) [inline plugins/org.eclipse.epsilon.workflow/ant/org/eclipse/epsilon/workflow/tasks/tasks.xml:1]: This change is unrelated to the purpose of the PR. Remove?

**Review by kolovos — COMMENTED** [review]
@arcanefoam Many thanks for the PR! Could you please prune it to exclude changes unrelated to Exeed (see detailed comments on individual files) and let me know when you've done this so that I can review?

File: plugins/org.eclipse.epsilon.workflow/ant/org/eclipse/epsilon/workflow/tasks/antlib.xml
Comment (arcanefoam) [reply plugins/org.eclipse.epsilon.workflow/ant/org/eclipse/epsilon/workflow/tasks/antlib.xml:1]: Fixed

File: plugins/org.eclipse.epsilon.workflow.emf/ant/org/eclipse/epsilon/workflow/tasks/emf/tasks.xml
Comment (arcanefoam) [reply plugins/org.eclipse.epsilon.workflow.emf/ant/org/eclipse/epsilon/workflow/tasks/emf/tasks.xml:1]: Fixed

File: plugins/org.eclipse.epsilon.workflow.emf/ant/org/eclipse/epsilon/workflow/tasks/emf/antlib.xml
Comment (arcanefoam) [reply plugins/org.eclipse.epsilon.workflow.emf/ant/org/eclipse/epsilon/workflow/tasks/emf/antlib.xml:1]: Fixed

File: plugins/.settings/org.eclipse.core.resources.prefs
Comment (arcanefoam) [reply plugins/.settings/org.eclipse.core.resources.prefs:1]: Fixed

File: plugins/.project
Comment (arcanefoam) [reply plugins/.project:1]: Fixed

File: plugins/org.eclipse.epsilon.dt.exeed/src/org/eclipse/epsilon/dt/exeed/ExeedImageTextProvider.java (hunk 1 above)
Comment (arcanefoam) [reply plugins/org.eclipse.epsilon.dt.exeed/src/org/eclipse/epsilon/dt/exeed/ExeedImageTextProvider.java:101]: Fixed

**Review by kolovos — APPROVED** [review]
Looks good to me!
